third-party/linshare/src/test/java/org/apache/james/linshare/LinshareExtension.java

```diff
@@ -109,6 +131,32 @@ public LinshareConfiguration configurationWithJwtFor(LinshareFixture.Credential
             .downloadShare(document.asString(), filename);
     }
 
+    @Override
+    public void beforeAll(ExtensionContext extensionContext){
+        TechnicalAccountCreationRequest technicalAccount = new TechnicalAccountCreationRequest(TECHNICAL_ACCOUNT.getUsername(),
```

Comment:
as we always create the same technical account, I suggest to have a factory method inside TechnicalAccountCreationRequest.

```java
class TechnicalAccountCreationRequest {
     private static final DEFAULT_USERNAME = ....
     static TechnicalAccountCreationRequest defaultAccount() {
             return ....;
     }
}
```
It helps to simplify beforeAll() and avoid confusion on the meaning of "technical@linshare.org", "true" and "DELEGATION"

Reply:
It's too soon to create a technical account ahead of every tests (which is necessary for the later task), having a method creating a technical account in the extension is enough for me

Reply:
I think it will need for both ( this task and later task).
@chibenwa what do you think?